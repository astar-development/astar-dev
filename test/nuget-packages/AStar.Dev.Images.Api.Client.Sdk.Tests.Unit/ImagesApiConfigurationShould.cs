using AStar.Dev.Images.Api.Client.Sdk.ImagesApi;
using AStar.Dev.Utilities;

namespace AStar.Dev.Images.Api.Client.Sdk;

public sealed class ImagesApiConfigurationShould
{
    [Fact]
    public void ReturnTheExpectedDefaultValue() =>
        new ImagesApiConfiguration { Scopes = [] }
            .ToJson()
            .ShouldMatchApproved();

    [Fact]
    public void ContainTheExpectedProperties() =>
        new ImagesApiConfiguration { Scopes = ["Mock Scope"], BaseUrl = new("https://doesnt.matter.com") }
            .ToJson()
            .ShouldMatchApproved();

    [Fact]
    public void ReturnTheExpectedSectionLocationValue() =>
        ImagesApiConfiguration.SectionLocation.ShouldBe("ApiConfiguration:ImagesApiConfiguration");
}
