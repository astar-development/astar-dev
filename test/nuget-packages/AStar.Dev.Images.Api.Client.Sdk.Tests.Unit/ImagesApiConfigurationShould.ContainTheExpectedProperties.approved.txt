{
  "baseUrl": "https://doesnt.matter.com",
  "scopes": [
    "Mock Scope"
  ]
}